components/data-services/org.wso2.carbon.dataservices.ui/src/main/resources/web/ds/addOperation.jsp

```diff
@@ -206,7 +206,7 @@
                 <th colspan="2"><fmt:message key="service.operations"/></th>
             </tr>
             </thead>
-            <form method="post" action="operationProcessor.jsp?action=<%=Encode.forHtmlAttribute(action)%>"
+            <form method="post" action="operationProcessor.jsp?action=<%=Encode.forHtml(action)%>"
```

Comment:
forUriComponent method should be used
https://owasp.github.io/owasp-java-encoder/encoder/apidocs/org/owasp/encoder/Encode.html#forUriComponent-java.lang.String-